OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewFragment.java

```diff
@@ -115,6 +121,40 @@ public void onPageScrollStateChanged(int state) {
         }
     }
 
+    public List<WhatsNewModel> getWhatsNewMessages(@NonNull String currBuildVersion,
+                                                   @NonNull List<WhatsNew> whatsNewList) {
+        final Version buildVersion;
+        Version whatsNewVersion;
+        try {
+            buildVersion = new Version(currBuildVersion);
+        } catch (ParseException e) {
+            logger.error(e, true);
+            return null;
+        }
+        for (WhatsNew whatsNew : whatsNewList) {
+            try {
+                whatsNewVersion = new Version(whatsNew.getVersion());
+            } catch (ParseException e) {
+                logger.error(e, true);
+                continue;
+            }
+            if (buildVersion.hasSameMajorMinorVersion(whatsNewVersion)) {
+                return filterAndroidMessages(whatsNew.getMessages());
+            }
+        }
+        return null;
+    }
+
+    public List<WhatsNewModel> filterAndroidMessages(@NonNull List<WhatsNewModel> messages) {
+        final List<WhatsNewModel> androidMessages = new ArrayList<>();
+        for (WhatsNewModel message : messages) {
+            if (message.getPlatforms().contains("android")) {
```

Comment:
"android" should be made a static final string. Might be a good idea to make this check case insensitive, thoughts?